### PR TITLE
Add installation instructions for Maxwell (DESY)

### PR DESCRIPTION
**Pull request by AngelFP**

### CODE REVIEW

File: doc/source/user_guide/installation_maxwell.rst [hunk 1]
```diff
@@ -1,2 +1,116 @@
 Installation on Maxwell (DESY)
 ------------------------------
+
+Log into a Maxwell display node
+
+.. code::
+
+    ssh username@max-display.desy.de
```
Comment (MaxThevenet) [inline doc/source/user_guide/installation_maxwell.rst:8]:
```suggestion
    ssh <username>@max-display.desy.de
```

File: doc/source/user_guide/installation_maxwell.rst [hunk 2]
```diff
@@ -1,2 +1,116 @@
...
+Install ``optimas``
+
+.. code::
+
+    pip install git+https://github.com/optimas-org/optimas.git
+
+
+Installing FBPIC and Wake-T (optional)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A common use case for ``optimas`` is to optimize the output of a plasma
```
Comment (MaxThevenet) [inline doc/source/user_guide/installation_maxwell.rst:63]:
```suggestion
A common use case for ``optimas`` is to optimize the output of a plasma acceleration
```

File: doc/source/user_guide/installation_maxwell.rst [hunk 3]
```diff
@@ -1,2 +1,116 @@
...
+.. code::
+
+    pip install openPMD-viewer
+
+
+Running an optimas job
+~~~~~~~~~~~~~~~~~~~~~~
+The following script can be used to submit an ``optimas`` job to Maxwell.
+This example sends the job to the ``maxgpu`` queue. It assumes that you
+need one ode with 4 GPUs (NVIDIA A100) and that your optimas script is called
```
Comment (MaxThevenet) [inline doc/source/user_guide/installation_maxwell.rst:94]:
```suggestion
need one node with 4 GPUs (NVIDIA A100) and that your optimas script is called
```

**Review by MaxThevenet — APPROVED** [review]
Looks great! See minor comments below.

**Comment by AngelFP** [comment]
Nice suggestions, thanks! All incorporated.

**Pull request merged**